### DISCUSSION TITLE
Which python versions are supported?

### DISCUSSION THREAD

**Discussion by Moon-Raven** [microsoft/pylance-release#1411] [Q&A]
Hello, is there a list of python versions that pylance supports? I am having issues with python 3.8 features being marked as problems inside VSCode.

**Comment by erictraut** [comment]
Pylance supports Python 3.0 and newer. It even includes support for all new features coming in Python 3.10.

Do you have a Python 3.8 interpreter selected? By default, Pylance assumes that it should restrict you to features available in the Python version you have selected. You can override this in a configuration file if you'd like.

If you'd like to share specific details about problems you're seeing, we may be able to better help diagnose.

**Reply by Moon-Raven** [reply]
I definitely had a different interpeter selected, compared to the one I was actually using. By telling VSCode which is the correct interpreter, PyLance started behaving as I expected. Thanks for the great extension and support!
